### DISCUSSION TITLE
Favorites: Show on map / Select on map

### DISCUSSION THREAD

**Discussion by Ciutnik** [devemux86/cruiser#296] [Q&A] [answered]
On the list of favorite points, after a long press, one of the items in the menu are `Show on map` and `Select on map`. 

In Cruiser 3.0.31 there is no difference in how these functions work. Both act as "Show". "Select" should bring up the point menu, but it doesn't. Can anyone else check how this works?

**Comment by devemux86** [comment]
@Ciutnik thanks for the report.

"Select on map" indeed does not currently work correctly (also in waypoints list).

It will work properly in the next Cruiser.

**Comment by devemux86** [comment, marked as answer]
Should be fixed in [Cruiser 3.0.32](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by umpfmpf** [comment]
I tried to quickly save a point as a favorite during a trip. it would be very nice to save it quickly with 2 or 3 clicks without having to stop and then finish editing (rename, move to folder) later.

**Reply by devemux86** [reply]
Indeed we should improve how to create favorites during navigation / follow, probably we need extra button.

Favorites can currently be added when we long press on map or use the menu favorites and the plus button.

**Reply by bios4** [reply]
Suggestion: provide a new button on screen during navigation/follow which immediately saves the current location (GPS coordinates, street name, ...) as a "temporary favorite" every time it is pressed.

Similar to the "highlight tag" in modern GoPro cameras, which add sort of a "bookmark" within a video recording as soon as the corresponding button is pressed.

These temporary favorites can then be updated (with proper name, favorite group etc.) after the trip.

The button should be configurable so that it is only shown for users which use this feature. ;-)

**Reply by devemux86** [reply]
@bios4 can you please create a new topic in "Ideas" with your suggestion so it doesn't get lost here?

---

I see we already have a topic for screenshots in navigation:
- https://github.com/devemux86/cruiser/discussions/95

**Reply by Ciutnik** [reply]
Here we also talked about the aptly named "WOW button". I know this solution from my old Garmin and I used it very often. In a split second, a waypoint named a sequence number was created. Time matters when moving 🙂

https://github.com/devemux86/cruiser/discussions/6#discussion-4713122

**Comment by Ciutnik** [comment]
While browsing a map with a lot of Favorites, it occurred to me that maybe a parameter specifying the zoom level at which a **given group** is visible would be useful? As is the case with via points and shaping points. In extreme settings it would be always or never – which is what we have now.

**Reply by devemux86** [reply]
As we start to populate the new favorites / routes / tracks we will probably see the need for more detailed management.

@Ciutnik can you please create a new topic with your idea?

**Comment by umpfmpf** [comment]
Would it be possible to search (hotel, restaurant...) around a travel destination or a defined point on the map? 
Would then be easy to plan a trip and choose a hotel as your destination.

**Reply by Ciutnik** [reply]
@umpfmpf 
This is more or less what this https://github.com/devemux86/cruiser/discussions/9#discussioncomment-7173338 proposal is about 🙂

**Reply by devemux86** [reply]
@umpfmpf 

Indeed, as @Ciutnik mentioned, in next Cruiser the search will be around the map center.
We can move the map anywhere and search for all points of interest in the specific region.
- https://github.com/devemux86/cruiser/discussions/9#discussioncomment-7177323

**Reply by umpfmpf** [reply]
That sounds very good. 
You are on a way to an very good navigation app.

**Comment by ramondevic** [comment]
I used to be able to move a favorite by clicking on it on the map and choosing the correct place where it should be, but now the "move" option is gone.
Could it be reactivated?
Why do some options that we used to have and were very useful disappear with the new updates? How do you decide what should continue or what should disappear?
Thanks.

**Reply by Ciutnik** [reply]
Try this, after long pressing Favorite on the map 🙂:

<img src="https://github.com/devemux86/cruiser/assets/122647782/b6c53836-6219-4a13-aa89-e84e7886df54" width="50%"/>

**Reply by ramondevic** [reply]
I've looked everywhere and the 032 (beta) doesn't have the option to move when it used to be there.
In addition, it is much more practical to do it from the map if you already have the favorite on the map, and not have to go to the list of favorites and then choose the option to view on the map, don't you think?

**Reply by Ciutnik** [reply]
But the above screenshot is after long pressing a point **on the map**! The only thing that has changed is that now "Move" is in the submenu.

**Reply by ramondevic** [reply]
OK, I see, after clicking on the Favorite on the map you have to click on the Favorite text again and then another menu opens with the option to Move

Thanks !

**Comment by ramondevic** [comment]
I uninstalled the ".32 (beta)" I had installed a couple of days ago and was very happy with it and reinstalled the .32 version of Google play.
For me, the option to "move" a favorite is essential.

**Comment by ramondevic** [comment]
Would it be possible that every time there is a new version (beta or official) put a list of the new functionalities and those that have disappeared?

Thanks.

**Comment by devemux86** [comment]
@ramondevic 

Some "related" submenus are grouped separately (they end with ...),
so that we don't have a very long list of actions.

It was announced with the other changes here:
- https://github.com/devemux86/cruiser/discussions/329#discussioncomment-7100352

If you are thinking about a more convenient implementation or different groups, we can discuss all alternatives.
One thing is for sure, all functions are always available. :slightly_smiling_face:

**Reply by ramondevic** [reply]
Yes, I already found it.

Thanks !
